styles/arch-wiki/catppuccin.user.css

```
@@ -429,6 +425,40 @@
     .mw-headline::before {
       color: @accent-color !important;
     }
+      
+    .cdx-button {
+      background-color: @base !important;
+      color: @text !important;
+      border-color: @base !important;
+    }
+      
+    .mw-body {
+      background-color: @mantle !important;
```

Content that is in the middle should be base following the style guide, perhaps make the header crust and the outer background mantle?
```suggestion
      background-color: @base !important;
```

---

i think thats a good idea yeah